### PR TITLE
update the translation of `Object.entries`

### PR DESCRIPTION
**Pull request by robert-jx**
### Description
update the translation of `Object.entries`

### Motivation
update the translation of `Object.entries`

### Additional details
none

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/object/entries/index.md [hunk 1]
```diff
@@ -5,91 +5,100 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/entries
 
 {{JSRef}}
 
-**`Object.entries()`** 方法返回一个给定对象自身可枚举属性的键值对数组，其排列与使用 {{jsxref("Statements/for...in", "for...in")}} 循环遍历该对象时返回的顺序一致（区别在于 for-in 循环还会枚举原型链中的属性）。
+**`Object.entries()`** 静态方法返回一个数组，由给定对象自身可枚举字符串键属性的键值对组成。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/entries/index.md:8]:
```suggestion
**`Object.entries()`** 静态方法返回一个数组，包含给定对象自有的可枚举字符串键属性的键值对。
```

File: files/zh-cn/web/javascript/reference/global_objects/object/entries/index.md [hunk 2]
```diff
@@ -5,91 +5,100 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/entries
...
 ### 参数
 
 - `obj`
-  - : 可以返回其可枚举属性的键值对的对象。
+  - : 一个对象。
 
 ### 返回值
 
-给定对象自身可枚举属性的键值对数组。
+一个由给定对象自身可枚举字符串键属性的键值对组成的数组。每个键值对都是一个包含两个元素的数组：第一个元素是属性的键（始终是字符串），第二个元素是属性值。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/entries/index.md:25]:
```suggestion
一个由给定对象自有的可枚举字符串键属性的键值对组成的数组。每个键值对都是一个包含两个元素的数组：第一个元素是属性的键（始终是字符串），第二个元素是属性值。
```

File: files/zh-cn/web/javascript/reference/global_objects/object/entries/index.md [hunk 3]
```diff
@@ -5,91 +5,100 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/entries
...
+
+// 其他基本类型没有自有属性
+console.log(Object.entries(100)); // []
 ```
 
-### 将`Object`转换为`Map`
+### 将 Object 转换成 Map
 
-{{jsxref("Map", "new Map()")}} 构造函数接受一个可迭代的`entries`。借助`Object.entries`方法你可以很容易的将{{jsxref("Object")}}转换为{{jsxref("Map")}}:
+{{jsxref("Map/Map", "Map()")}} 构造函数接受 `entries` 的可迭代项。使用 `Object.entries`，你可以很容易地从 {{jsxref("Object")}} 转换成 {{jsxref("Map")}}：
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/entries/index.md:78]:
```suggestion
{{jsxref("Map/Map", "Map()")}} 构造函数接受一个`entries` 可迭代对象。使用 `Object.entries`，你可以很容易地将 {{jsxref("Object")}} 转换成 {{jsxref("Map")}}：
```

**Review by JasonLamv-t — COMMENTED** [review]
Some suggestions.

File: files/zh-cn/web/javascript/reference/global_objects/object/entries/index.md [hunk 4]
```diff
@@ -5,91 +5,100 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/entries
...
+
+// 其他基本类型没有自有属性
+console.log(Object.entries(100)); // []
 ```
 
-### 将`Object`转换为`Map`
+### 将 Object 转换成 Map
 
-{{jsxref("Map", "new Map()")}} 构造函数接受一个可迭代的`entries`。借助`Object.entries`方法你可以很容易的将{{jsxref("Object")}}转换为{{jsxref("Map")}}:
+{{jsxref("Map/Map", "Map()")}} 构造函数接受一个`entries` 可迭代对象。使用 `Object.entries`，你可以很容易地将 {{jsxref("Object")}} 转换成 {{jsxref("Map")}}：
```
Comment (skyclouds2001) [inline files/zh-cn/web/javascript/reference/global_objects/object/entries/index.md:78]:
```suggestion
{{jsxref("Map/Map", "Map()")}} 构造函数接受一个 `entries` 可迭代对象。使用 `Object.entries`，你可以很容易地将 {{jsxref("Object")}} 转换成 {{jsxref("Map")}}：
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**